WORKSPACE

```diff
@@ -357,9 +330,9 @@ container_pull(
 # TODO build fedora_with_test_tooling for multi-arch
 container_pull(
     name = "fedora_with_test_tooling",
-    digest = "sha256:14193941e1fe74f2189536263c71479abbd296dc93b75e8a7f97f0b31e78b71e",
-    registry = "quay.io",
-    repository = "kubevirtci/fedora-with-test-tooling",
+    digest = "sha256:d12cf2d45c8fb1b07d36d29450b73cf78050abc65b42f851ca452ca1caf9ef08",
+    registry = "ghcr.io",
+    repository = "vasiliy-ul/fedora-with-test-tooling",
```

Comment:
We need to make sure this PR is not merged with these.

Reply:
Right. We discussed that topic briefly with @rmohr in [slack](https://kubernetes.slack.com/archives/C0163DT0R8X/p1639467201175900). I pushed the image to a temporary registry in order to work on the PR and fix the issues. At the end it should be pushed to quay.io/kubevirt but I think I will probably need some help with that (from someone with proper permissions).